lib/Browser.js

```diff
@@ -45,8 +89,24 @@ class Browser extends EventEmitter {
    */
   async newPage() {
     const {targetId} = await this._connection.send('Target.createTarget', {url: 'about:blank'});
-    const client = await this._connection.createSession(targetId);
-    return await Page.create(client, this._ignoreHTTPSErrors, this._appMode, this._screenshotTaskQueue);
+    const {targetInfo} = await this._connection.send('Target.getTargetInfo', {targetId});
+    const page = await this._targetForInfo(targetInfo).page();
+    await page.goto('about:blank');
+    return page;
+  }
+
+  /**
+   * @return {Array<Target>}
+   */
+  targets() {
+    return Array.from(this._targets.values());
+  }
+
+  /**
+   * @return {Array<Pages>}
```

Comment:
Promise type declaration is missing

I also like `async` methods since they make it really obvious that the method should be awaited. Let's make `pages` async!